src/measurement/position_measurement.js

```diff
@@ -427,7 +427,7 @@ export function coordsChar(cm, x, y) {
 function wrappedLineExtent(cm, lineObj, preparedMeasure, y) {
   let measure = ch => intoCoordSystem(cm, lineObj, measureCharPrepared(cm, preparedMeasure, ch), "line")
   let end = lineObj.text.length
-  let begin = findFirst(ch => measure(ch).bottom < y, end - 1, 0) + 1
+  let begin = findFirst(ch => measure(ch).bottom <= y, end - 1, -1) + 1
```

Comment:
That should do it, shouldn't it?
```javascript
let begin = findFirst(ch => measure(ch - 1).bottom <= y, end, 0)
```
I find the current version to be a bit closer to the actual intent, though. It's the character after (`+ 1`) the last character (`first` from `length - 1`) whose bottom is not `<= y`. The simpler version looks simpler, but doesn't convey any meaning. As for off-by-one errors: I found tests to be a good solution to that problem :)